NuGet.config

```
@@ -5,6 +5,7 @@
   </solution>
   <packageSources>
     <clear />
+    <add key="darc-pub-dotnet-maintenance-packages-ab95a1f1" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-maintenance-packages-ab95a1f1/nuget/v3/index.json" />
```

Agh this was the problem @michaelgsharp 
```suggestion
    <add key="darc-pub-dotnet-maintenance-packages-ab95a1f1" value="https://dnceng.pkgs.visualstudio.com/public/_packaging/darc-pub-dotnet-maintenance-packages-ab95a1f1/nuget/v3/index.json" />
```